lib/src/selectors/date_picker.dart

```diff
@@ -59,6 +60,8 @@ class MacosDatePicker extends StatefulWidget {
   /// Defaults to `DateTime.now()`.
   final DateTime? initialDate;
 
+  final bool? startWeekOnMonday;
```

Comment:
Please add some documentation describing this property's purpose. It may be intuitive for some, but for others it may not be.